src/components/Datepicker/DatepickerRangeInput.tsx

```diff
@@ -162,6 +162,12 @@ interface DatepickerRangeInputProps extends MarginProps, WidthProps {
      * The id to be assigned to the end date input
      */
     endInputId?: string;
+    /**
+     * Determines the variant
+     * @value `'compact'` displays only a single month
+     * @default 'normal'
+     */
+    variant?: 'compact' | 'normal';
```

Comment:
Usually `variant` describes the difference between a bottom-lined and a boxed input (at the moment we only have boxed date picker versions). Have you considered exposing the `numberOfMonths` field instead?

Reply:
I thought about it, but it seemed more technical and on the path of being a workaround rather than a solution. Plus, that opens up the pandoras box of being able to have 3 months displayed, 4 months displayed etc, which will require changes in the component logic to not break anything, imo.

If you are strongly opinionated about this, maybe we can expose something like `displaySingleMonth: boolean` or other alternatives, but `compact` seemed much more developer friendly :) 

> I know the PR is merged already, but happy to keep the conversation going.